docs/resources/Guild_Scheduled_Event.md

```diff
@@ -101,35 +157,102 @@ Modify a guild scheduled event. Returns the modified [guild scheduled event](#DO
+| privacy_level         | [privacy level](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-privacy-level)     | the privacy level of the scheduled event                                                   |
+| scheduled_start_time  | ISO8601 timestamp                                                                                                           | the time to schedule the scheduled event                                                   |
+| scheduled_end_time    | ISO8601 timestamp                                                                                                           | the time when the scheduled event is scheduled to end                                      |
+| description           | string                                                                                                                      | the description of the scheduled event                                                     |
+| entity_type *         | [event entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-types)  | the entity type of the scheduled event                                                     |
+| status                | [event status](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-status)             | the status of the scheduled event                                                          |
+
+\* If updating `entity_type` to `EXTERNAL`, `channel_id` [must be set to null](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-field-requirements-by-entity-type)
 
 ## Delete Guild Scheduled Event % DELETE /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/scheduled-events/{guild_scheduled_event.id#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object}
 
 Delete a guild scheduled event. Returns a `204` on success.
 
 ## Get Guild Scheduled Event Users % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/scheduled-events/{guild_scheduled_event.id#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object}/users
 
-Get a list of users RSVP'd to the guild scheduled event. Returns a list of [user](#DOCS_RESOURCES_USER/user-object) objects on success with an optional `guild_member` property for each user if `with_member` query parameter is passed in.
+> warn
+> A breaking change was introduced for this endpoint on Thursday Nov 18, 2021 after the initial publication of this documentation in which the return type was changed in response to developer feedback. We apologize for the inconvenience and additional work this creates for developers.
+
+Get a list of users subscribed to a guild scheduled event. Returns a list of [guild scheduled event user](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-user-object) objects on success. Guild member data, if it exists, is included if the `with_member` query parameter is set.
```

Comment:
With some testing yesterday, I found out that this endpoint is paginated and respected the `after={user_id}` query parameter, will that still be the case? If so, that should be documented.

Reply:
yes, both `before` and `after` have been part of the api implementation so should be functional but we've been holding off on officially adding this until post launch since we don't currently use these internally. But at this point I think some libraries are starting to use it already anyway so could just do a bit more testing and prob include this as part of this docs update